lib/yggdrasil/plug/SkipIfAuthenticated.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Yggdrasil.Plug.SkipIfAuthenticated do
+  import Plug.Conn
+  import Yggdrasil.Router.Helpers
+  import Phoenix.Controller
+
+  def init(default), do: default
+
+  def call(conn, default) do
```

Comment:
Is it possible to test this? I saw that the plug library has a bit of documentation in their [readme](https://github.com/elixir-lang/plug#testing-plugs)